src/main/kotlin/backend/GenMutator.kt

```diff
@@ -33,4 +33,6 @@ class GenMutator(private val config: Config) {
 
     return Genome(newGenes, random.nextInt(genomeLength))
   }
+
+  fun random(): Genome = Genome(List(config.genomeLength) { Gen.random(random) }, random.nextInt(config.genomeLength))
```

Comment:
Should `GenMutator` provide a random Genome? Change the class name. `GenomeManage`, `GenomeProvider`, `GenomeOperator` or sth better